src/pivot-table/hooks/use-column-width.ts

```diff
@@ -28,6 +28,11 @@ export interface GetHeaderCellsIconsVisibilityStatus {
   };
 }
 
+interface LeftGridColumnWidths {
+  widths: number[];
+  originalWidths: number[];
```

Comment:
I found `originalWidths` to be hard to figure out why it was needed. Before I realized that it's there to get the full width of the columns. Maybe rename it to `fullWidths`? As the `widths` is "capped" at the 75% rule right?

Reply:
I thought of this a bit more and came up with more clearer names, now we have:
`actualColumnWidths`: which refers to the base/original width per column that we receive from sense-client 
`modifiedColumnWidth`:  which reflects the widths per column if we modify its value (currently only being used to calculate the left grid width)

Hope it sounds well